CONTRIBUTORS

```diff
@@ -1,6 +1,12 @@
 # Please keep the list sorted.
 
+Alen Vrečko 
+Alexey Volkov <lehvolk@yandex.com>
 Andrei Pangin <noreply@pangin.pro>
+Leonid Talalaev <leonid.talalaev@gmail.com>
+MacX19 
```

Comment:
It would be nice to add e-mail if possible.

Reply:
`Maksim Ponomarev <macx@mail.ru>`